adls-plugins/src/main/java/co/cask/hydrator/plugin/sink/ADLSBatchSink.java

```diff
@@ -140,6 +166,12 @@ protected void validate() {
       if (!containsMacro("path") && !path.startsWith("adl://")) {
         throw new IllegalArgumentException("Path must start with adl:// for ADLS input files.");
       }
+      if (TEXT.equals(outputFormat) && textDelimiter == null) {
```

Comment:
validate should be called in configurePipeline as well, as these types of errors should be caught during pipeline deployment and not at runtime.